components/user_settings/notifications/user_settings_notifications.tsx

```diff
@@ -433,7 +464,7 @@ export default class NotificationsTab extends React.PureComponent {
             );
         }
 
-        let describe = '';
+        let describe: string | JSX.Element = '';
```

Comment:
I think `React.ReactNode` might be the correct type here since that's what we usually use as the type of `this.props.children` and this would work the same